### ISSUE TITLE
Remove google play dependency

### ISSUE THREAD

**Issue by lordfolken** [Aptoide/aptoide-client#8]
Hello, 

Sadly aptoide complains that no google play libraries are installed. It still works, but is unable to download any applications.

**Comment by Rmota89** [comment]
Hi, 

Can you tell us what device/rom are you using?

**Comment by lordfolken** [comment]
Hi, 

Sure:

Device model: GT-I9100
Android Version: 6.0.1
CynaogenMod version:
13.0-20160409-NIGHTLY-i9100
Android security patch level: April 2, 2016

There are no google apps / play libraries installed on the device. (Also not opengapps.org) or similar.
Hence the motivation for using aptoide. Aptoide light works fine for me. 

I also tried the root mode of aptoide, but that doesn't seem to make a difference. 

any more info you want, e.g. logcat?

 Cheers,
- Folken

**Comment by lordfolken** [comment]
The aptoide version is 7.1.0.7. Permissions granted: Storage.

**Comment by Rmota89** [comment]
Hi, 

When do you receive that dialog about Google Services? When you try to install an App? 

Send us a logcat file too please. 

Best Regards,

**Comment by henriquedesousa** [comment]
I've encountered this issue. It seems that, when you have an outdated _Google Play Services_ app installed on your device, when you go to the Login screen it will trigger the following dialog, although the app will work just fine. 
![screenshot_2016-04-24-19-10-52](https://cloud.githubusercontent.com/assets/12630569/14769896/09f97fe4-0a5b-11e6-9693-582210d1129f.png)

**Comment by Rmota89** [comment]
Yes, that one we've already identified, but we're not able to reproduce the
one that lordfolken reported.

Henrique de Sousa notifications@github.com escreveu no dia domingo,
24/04/2016 às 20:28:
